Expecto/Expecto.fsproj

```diff
@@ -151,49 +151,6 @@
     </When>
   </Choose>
   <Choose>
-    <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And ($(TargetFrameworkVersion) == 'v2.0' Or $(TargetFrameworkVersion) == 'v3.0')">

```

Comment:
Oh I see. But we should be targeting 4.6.1, not 4.0. v4.0 is only a thing [here in albacore](https://github.com/Albacore/albacore/issues/148)